polynote-frontend/polynote/state/preferences.ts

```diff
@@ -70,6 +76,12 @@ export const ViewPrefsHandler = new LocalStorageHandler<ViewPreferences>("ViewPr
         collapsed: false,
     }
 });
+export const LeftBarPrefsHandler = new LocalStorageHandler<LeftBarPreferences>("LeftBarPreferences", {
```

Comment:
Realistically the left bar state storage should be part of the `ViewPreferences` in `LocalStorage`, but it was a non-trivial effort to get it to handle schema changes like `UserPreferences` does - I figured it's better to leave that for a future PR because I was running into a few different issues.